R/timebased.R

```diff
@@ -128,17 +131,17 @@ timebased <- function(data_analyze, date_variable, time_unit="auto", outdir) {
           scale_fill_brewer(palette='Paired', label=abbr) +
           theme_minimal() +
           labs(x = var$Variable, y = "Rows", fill=varName) +
-          ggtitle(paste0("Evolution of variable ", varName)) +
+          ggtitle(paste("Evolution of variable", varName)) +
```

Comment:
I see you changed some of the pastes to paste0 and paste0 to paste, i'm ignorant as to why, can you explain me? Thanks!

Reply:
This is more of a stylistic point than anything--it doesn't really matter.  `paste0(...)` is basically `paste(..., sep = "")`.  Here `paste` saves 2 keystrokes (the zero and the space at the end of the string; obviously doesn't matter at all), but in my opinion is more idiomatic since `paste0` basically exists to save writing `sep = ""`.  Since we want a space separator anyway, there's really no need for `paste0`.

Reply:
Great! Thanks for explaining